pkg/gossip/gossip.go

```diff
@@ -521,6 +521,12 @@ func (g *Gossip) GetAddresses() []util.UnresolvedAddr {
 	return append([]util.UnresolvedAddr(nil), g.addresses...)
 }
 
+func (g *Gossip) GetBootstrapAddresses() map[util.UnresolvedAddr]roachpb.NodeID {
```

Comment:
`GetBootstrapAddresses` is used to get list of known node address in cluster as a source of known nodes.